### ISSUE TITLE
npm install fail - Failed at the fusejs@1.3.1 install script 'node-gyp rebuild'.

### ISSUE THREAD

**Issue by ecchi** [thejinx0r/node-gdrive-fuse#86]
```
npm WARN deprecated MD5@1.3.0: deprecated, use lowercase 'md5@2.x' from now on
...
> fusejs@1.3.1 install /Users/ecchi/Code/node-gdrive-fuse/node_modules/fusejs
> node-gyp rebuild

../src/forget_data.cc:40:85: error: member access into incomplete type 'struct fuse_forget_data'
                info.GetReturnValue().Set( Nan::New<Integer>(static_cast<uint32_t>(forget_data->fd->ino)));
                                                                                                  ^
../src/forget_data.h:19:20: note: forward declaration of 'NodeFuse::fuse_forget_data'
            struct fuse_forget_data *fd;
                   ^
../src/forget_data.cc:40:87: error: expected ')'
                info.GetReturnValue().Set( Nan::New<Integer>(static_cast<uint32_t>(forget_data->fd->ino)));
                                                                                                    ^
../src/forget_data.cc:40:69: note: to match this '('
                info.GetReturnValue().Set( Nan::New<Integer>(static_cast<uint32_t>(forget_data->fd->ino)));
                                                                                  ^
../src/forget_data.cc:47:86: error: member access into incomplete type 'struct fuse_forget_data'
                info.GetReturnValue().Set( Nan::New<Integer>( static_cast<uint32_t>(forget_data->fd->nlookup)) );
                                                                                                   ^
../src/forget_data.h:19:20: note: forward declaration of 'NodeFuse::fuse_forget_data'
            struct fuse_forget_data *fd;
                   ^
../src/forget_data.cc:47:88: error: expected ')'
                info.GetReturnValue().Set( Nan::New<Integer>( static_cast<uint32_t>(forget_data->fd->nlookup)) );
                                                                                                     ^
../src/forget_data.cc:47:70: note: to match this '('
                info.GetReturnValue().Set( Nan::New<Integer>( static_cast<uint32_t>(forget_data->fd->nlookup)) );
                                                                                   ^
4 errors generated.
make: *** [Release/obj.target/fusejs/src/forget_data.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 11.4.2
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/ecchi/Code/node-gdrive-fuse/node_modules/fusejs
gyp ERR! node -v v5.0.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok 
...
npm WARN EPACKAGEJSON GDriveF4JS@0.6.3 No license field.
npm ERR! Darwin 11.4.2
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v5.0.0
npm ERR! npm  v3.3.6
npm ERR! code ELIFECYCLE
...
npm ERR! fusejs@1.3.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fusejs@1.3.1 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the fusejs package,

```

same output with node 0.10.40

**Comment by thejinx0r** [comment]
What version of osxfuse are you using?

On Sat, Oct 31, 2015 at 1:17 PM ecchi <notifications@github.com> wrote:

> npm WARN deprecated MD5@1.3.0: deprecated, use lowercase 'md5@2.x' from now on
> ...
> > fusejs@1.3.1 install /Users/ecchi/Code/node-gdrive-fuse/node_modules/fusejs
> > node-gyp rebuild
>
> ../src/forget_data.cc:40:85: error: member access into incomplete type 'struct fuse_forget_data'
>                 info.GetReturnValue().Set( Nan::New<Integer>(static_cast<uint32_t>(forget_data->fd->ino)));
>                                                                                                   ^
> ../src/forget_data.h:19:20: note: forward declaration of 'NodeFuse::fuse_forget_data'
>             struct fuse_forget_data *fd;
>                    ^
> ../src/forget_data.cc:40:87: error: expected ')'
>                 info.GetReturnValue().Set( Nan::New<Integer>(static_cast<uint32_t>(forget_data->fd->ino)));
>                                                                                                     ^
> ../src/forget_data.cc:40:69: note: to match this '('
>                 info.GetReturnValue().Set( Nan::New<Integer>(static_cast<uint32_t>(forget_data->fd->ino)));
>                                                                                   ^
> ../src/forget_data.cc:47:86: error: member access into incomplete type 'struct fuse_forget_data'
>                 info.GetReturnValue().Set( Nan::New<Integer>( static_cast<uint32_t>(forget_data->fd->nlookup)) );
>                                                                                                    ^
> ../src/forget_data.h:19:20: note: forward declaration of 'NodeFuse::fuse_forget_data'
>             struct fuse_forget_data *fd;
>                    ^
> ../src/forget_data.cc:47:88: error: expected ')'
>                 info.GetReturnValue().Set( Nan::New<Integer>( static_cast<uint32_t>(forget_data->fd->nlookup)) );
>                                                                                                      ^
> ../src/forget_data.cc:47:70: note: to match this '('
>                 info.GetReturnValue().Set( Nan::New<Integer>( static_cast<uint32_t>(forget_data->fd->nlookup)) );
>                                                                                    ^
> 4 errors generated.
> make: *** [Release/obj.target/fusejs/src/forget_data.o] Error 1
> gyp ERR! build error
> gyp ERR! stack Error: `make` failed with exit code: 2
> gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
> gyp ERR! stack     at emitTwo (events.js:87:13)
> gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
> gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
> gyp ERR! System Darwin 11.4.2
> gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
> gyp ERR! cwd /Users/ecchi/Code/node-gdrive-fuse/node_modules/fusejs
> gyp ERR! node -v v5.0.0
> gyp ERR! node-gyp -v v3.0.3
> gyp ERR! not ok
> ...
> npm WARN EPACKAGEJSON GDriveF4JS@0.6.3 No license field.
> npm ERR! Darwin 11.4.2
> npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
> npm ERR! node v5.0.0
> npm ERR! npm  v3.3.6
> npm ERR! code ELIFECYCLE
> ...
> npm ERR! fusejs@1.3.1 install: `node-gyp rebuild`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the fusejs@1.3.1 install script 'node-gyp rebuild'.
> npm ERR! This is most likely a problem with the fusejs package,
>
>
> same output with node 0.10.40
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thejinx0r/node-gdrive-fuse/issues/86>.
>
-- 
Eric

**Comment by ecchi** [comment]
2.8.2

**Comment by thejinx0r** [comment]
Resolved in c4milo/fusejs#27
